app/queries/developer_query.rb

```diff
@@ -15,6 +15,7 @@ def initialize(options = {})
     @role_levels = options.delete(:role_levels)
     @include_not_interested = options.delete(:include_not_interested)
     @search_query = options.delete(:search_query)
+    @search_type = options.delete(:search_type)
```

Comment:
Let's move all of these changes to a new model instead of trying to fit them inside of this object.